### ISSUE TITLE
GUI: blocks update too slow

### ISSUE THREAD

**Issue by IsabellaRey** [ISISComputingGroup/IBEX#1607]
When a disconnected block becomes connected, the block display takes a long time to update. One ends up restarting the GUI to see its connected state. It should detect the change faster

**Comment by AdrianPotter** [comment]
Believe fixed with #1614

**Comment by AdrianPotter** [comment]
False alarm. Seems better after #1614 but still too slow for my tastes

**Comment by AdrianPotter** [comment]
Seems to be working well with version 2.2.0. Perhaps fixed by #1774? Setting a 0 label. Please could someone retest and close if it is fixed? Alternatively, put it back in the backlog

**Comment by mattclarke** [comment]
Closed as superseded by https://github.com/ISISComputingGroup/IBEX/issues/2162
